docs/dcp2_system_design.rst

```diff
@@ -1199,6 +1222,72 @@ for a project with one non-stratified matrix.
    that file may have a more up-to-date list
 
 
+Describing CGMs as analysis files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The mechanism defined in `Describing CGMs as supplementary files`_ is
+deprecated. Instead of representing a contributor-generated matrix (CGM) as a
+stand-alone supplementary file, it is described more accurately as an
+``analysis_file`` entity that is an output of an ``analysis_process`` entity.
+
+Information about the provenance of the CGMs for a particular project may be
+limited. The DCP/2 may not be in possession of intermediate artifacts like the
+sequence data or any information about those artifacts because, for example,
+they are subject to access restrictions the DCP/2 does not yet have the
+capabilities to enforce. Even if the DCP/2 is in possession of intermediate
+artifacts and knows their provenance, their bearing on the contributed
+matrices may not have been supplied.
+
+Either way, the subgraph(s) containing CGM analysis files may need to be an
+approximation. If they are, the ``analysis_process`` entity that lists one or
+more CGMs as its output will not have any sequence files or other intermediate
+matrice files as inputs — as they would in the case of `DCP/2-generated
+matrices`_ — but instead directly refer to biomaterial entities like
+``specimen_from_organism``, ``cellline`` or ``organoid`` as its input,
+skipping the sequence files and any intermediate matrices.
+
+Consequently, the Azul indexer extends its definition of *sample* to include
+not only biomaterials that are used as sequencing inputs but also any
+biomaterials linked directly to such an approximate `analysis_process` entity.
+
+If information is available about the protocols used in the experiment
+yielding the CGMs, that information is captured in protocol entities such as
+``sequencing_protocol`` or ``library_construction_protocol`` and those
```

Comment:
should be ``library_preparation_protocol``

Reply:
Addressed.